LoopKitUI/OnboardingUI.swift

```diff
@@ -136,6 +136,17 @@ public enum StudyProduct: String, CaseIterable {
     case none
     case studyProduct1
     case studyProduct2
+    
+    public func filtered(scenarioURLs: [URL]) -> [URL] {
```

Comment:
Sorry I missed the earlier PR where `SudyProduct` ended up here, but LoopKit should be independent of special Tidepool cases; it should be restricted (as much as possible) to providing general purpose tools that could be used by other diabetes focused app.

Reply:
Yeah, I gave @Camji55 the same feedback on that previous PR but after discussion it seemed this was the best place for it.

@ps2 do you have something in mind?

Reply:
Made some changes here